test/capi/capiLottie.cpp

```diff
@@ -50,6 +50,12 @@ TEST_CASE("Lottie Slot", "[capiLottie]")
     REQUIRE(tvg_picture_load(picture, TEST_DIR"/lottieslot.json") == TVG_RESULT_SUCCESS);
     REQUIRE(tvg_lottie_animation_override(animation, slotJson) == TVG_RESULT_SUCCESS);
 
+    //Slot revert
+    REQUIRE(tvg_lottie_animation_override(animation, nullptr) == TVG_RESULT_SUCCESS);
```

Comment:
probably, you can add this as well before the first slot overriding call (45 line?)